SConstruct

```diff
@@ -393,16 +393,20 @@ def config_base(env):
         env.Append(CPPPATH=[os.path.join(profile_jemalloc, 'include')])
         env.Append(LINKFLAGS=['-Wl,-rpath,' + os.path.join(profile_jemalloc, 'lib')])
 
-def add_static_libs(env, libs):
+def add_static_libs(env, static_libs, dyn_libs=None):
     if not 'HASSTATICLIBS' in env:
         env['HASSTATICLIBS'] = True
         env['STATICLIBS'] = ''
-        env.Replace(LINKCOM=env['LINKCOM'] + " -Wl,-Bstatic $STATICLIBS")
-    c = env['STATICLIBS']
-    if c == None: c = ''
-    for f in libs:
-        c += ' -l' + f 
-    env['STATICLIBS'] = c
+        env['DYNAMICLIBS'] = ''
```

Comment:
I append to those variables, and they need to be initialized as strings. Line 405 gets the old value, line 409 assigns the new value. Both variables will be empty at line 399, but I can add a check for that assumption.